pkg/check/check.go

```diff
@@ -11,7 +11,9 @@ import (
 func Stale(lastVersionCheck string, dur string) bool {
 	d, err := time.ParseDuration("-" + dur)
 	if err != nil {
-		return false
+		// if there is no duration provided, then we should presume the loading of
+		// remote configuration failed and that we should retry that operation.
+		return true
 	}
 
 	if t, _ := time.Parse(time.RFC3339, lastVersionCheck); !t.Before(time.Now().Add(d)) {
```

Comment:
It also occurs to me that we're not checking the error here either. A scenario could be that we get this far but have a blank lastVersionCheck?

Reply:
We now won't get here because we have the additional check for:

```go
if file.CLI.LastChecked == "" {
```

Before we check stale.